hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1797,6 +1797,40 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.s3g.secret.http.enabled</name>
+    <value>false</value>
+    <tag>OZONE, S3GATEWAY</tag>
+    <description>The boolean which enables the Ozone S3Gateway Secret endpoint.
+    </description>
+  </property>
+  <property>
+    <name>ozone.s3g.secret.http.auth.type</name>
+    <value>simple</value>
+    <tag>S3GATEWAY, SECURITY, KERBEROS</tag>
+    <description> simple or kerberos. If kerberos is set, Kerberos SPNEOGO
+      will be used for http authentication.
+    </description>
+  </property>
+  <property>
+    <name>ozone.s3g.secret.http.auth.kerberos.principal</name>
+    <value>HTTP/_HOST@REALM</value>
```

Comment:
We need a different default value for this property because it's the same as "ozone.s3g.http.auth.kerberos.principal".